### PR TITLE
macOS/Mac Catalyst dev tools update

### LINKED ISSUE #28109: Crosslink new macOS Mac Catalyst/Safari guidance
Per https://github.com/dotnet/docs-maui/pull/1197.

Thanks @davidbritch ... I think I can pull the following line ...

> Using browser developer tools with Mac Catalyst apps isn't currently supported. You may wish to run the \[iOS](https://learn.microsoft.com/en-us/aspnet/core/blazor/hybrid/developer-tools?pivots=ios&view=aspnetcore-7.0) application instead, as the behavior is similar between iOS and Mac Catalyst.

... from https://learn.microsoft.com/en-us/aspnet/core/blazor/hybrid/developer-tools?view=aspnetcore-7.0&pivots=macos

... and replace it with a crosslink to the new guidance that you placed ... something like ...

> To inspect a Blazor Hybrid app on Mac Catalyst, see \[Host a Blazor web app in a .NET MAUI app using BlazorWebView](/dotnet/maui/user-interface/controls/blazorwebview#inspect-a-blazor-hybrid-app-on-mac-catalyst).

Does that seem sufficient to you?

The alternative is that I copy and paste the section, but that makes for the possibility that something would shift in a future release that wouldn't get picked up on this end for a stale bit of guidance 😈.

If we go with that line ☝️, I'll mark the PR ***DRAFT*** and keep an eye on the live doc over there to make sure it doesn't merge until your new guidance goes live before merging the PR.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 48e7c079-a7e4-e0a4-ed19-935436113b70
* Version Independent ID: 48e7c079-a7e4-e0a4-ed19-935436113b70
* Content: [Use browser developer tools with ASP.NET Core Blazor Hybrid](https://learn.microsoft.com/en-us/aspnet/core/blazor/hybrid/developer-tools?view=aspnetcore-7.0)
* Content Source: [aspnetcore/blazor/hybrid/developer-tools.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/blazor/hybrid/developer-tools.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-blazor**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #28109

David ... This seems to be headed in the right direction, but a few things cropped up ...

1. Instead of **Develop** > **Simulator** > **0.0.0.0** in Safari, it now shows some kind of machine ID or other ID there. It's like "V43TSD874E1" ... something like that. When you roll over that, you can see the **0.0.0.0**, which spawns the Web Inspector. I wasn't sure how to describe what that is in the doc other than calling it a "machine ID." Is that what that is?
2. I kept the guidance about "... the BlazorWebView is highlighted in blue in the macOS simulator when the correct **0.0.0.0** entry is selected ..." That doesn't seem correct tho for macOS/Mac Catalyst. I didn't see anything highlighted in the macOS simulator. Should that bit be stricken, or did I look in the wrong place?

I'm going to step out to the 🏋️💪🏃 for a short bit, and I'll be back to walk the steps again to double-check everything and react to your feedback.

### CODE REVIEW

File: aspnetcore/blazor/hybrid/developer-tools.md [hunk 1]
```diff
@@ -86,20 +93,42 @@ To use Safari developer tools with an iOS app:
...
+</PropertyGroup>
+```
 
-<!--
 To use Safari developer tools with a macOS app:
 
 1. Open desktop Safari.
-1. Select the **Preferences** > **Advanced** > **Show Develop** menu command.
+1. Select the **Safari** > **Preferences** > **Advanced** > **Show Develop menu in the menu bar** checkbox.
 1. Run the .NET MAUI Blazor app in the macOS simulator.
```
Comment (davidbritch) [inline aspnetcore/blazor/hybrid/developer-tools.md:127]: It's not running in a simulator. It's just running in macOS.

**Review by davidbritch — APPROVED** [review]
Approved, but with one specific comment that needs addressing.

**Comment by davidbritch** [comment]
> 1. Instead of **Develop** > **Simulator** > **0.0.0.0** in Safari, it now shows some kind of machine ID or other ID there. It's like "V43TSD874E1" ... something like that. When you roll over that, you can see the **0.0.0.0**, which spawns the Web Inspector. I wasn't sure how to describe what that is in the doc other than calling it a "machine ID." Is that what that is?

This is exactly the behaviour I experienced yesterday, after first enabling Safari developer tools. However, today (presumably post reboot) I now see a sensible name for my machine:

image

In addition, if you want to link to help docs for Safari Developer Tools, the link is https://support.apple.com/guide/safari-developer/dev073038698/mac. This link makes use of the term "remote inspection targets" and "listed devices" to refer to devices that can be inspected.

**Comment by davidbritch** [comment]
> 2\. I kept the guidance about "... the BlazorWebView is highlighted in blue in the macOS simulator when the correct **0.0.0.0** entry is selected ..." That doesn't seem correct tho for macOS/Mac Catalyst. I didn't see anything highlighted in the macOS simulator. Should that bit be stricken, or did I look in the wrong place?

It should just be "in macOS", not the "macOS simulator" - it's not running in a simulator. Your description is largely what I see - the app window containing the BlazorWebView is highlighted in blue when I select the menu item.

File: aspnetcore/blazor/hybrid/developer-tools.md [hunk 2]
```diff
@@ -86,20 +93,42 @@ To use Safari developer tools with an iOS app:
...
-1. Return to desktop Safari. Select **Develop** > **Simulator** > **0.0.0.0**. If multiple entries for **0.0.0.0** are present, select the entry that highlights the <xref:Microsoft.AspNetCore.Components.WebView.Maui.BlazorWebView>. The <xref:Microsoft.AspNetCore.Components.WebView.Maui.BlazorWebView> is highlighted in blue in the macOS simulator when the correct **0.0.0.0** entry is selected.
+1. Return to Safari. Select **Develop** > **{MACHINE ID}** > **0.0.0.0**, where the `{MACHINE ID}` placeholder is the machine's identifier. If multiple entries for **0.0.0.0** are present, select the entry that highlights the <xref:Microsoft.AspNetCore.Components.WebView.Maui.BlazorWebView>. The <xref:Microsoft.AspNetCore.Components.WebView.Maui.BlazorWebView> is highlighted in blue in the macOS simulator when the correct **0.0.0.0** entry is selected.
```
Comment (davidbritch) [inline aspnetcore/blazor/hybrid/developer-tools.md:128]: "in the macOS simulator" > "in macOS"

File: aspnetcore/blazor/hybrid/developer-tools.md (hunk 1 above)
Comment (guardrex) [reply aspnetcore/blazor/hybrid/developer-tools.md:127]: This merely occurred from coping over content from the iOS content, where a simulator is run. I'll fix it up on the commit.

**Comment by guardrex** [comment]
> presumably post reboot

No 🎲🎲 for me on a reboot. It's still not a plain machine name. I did find out what that is tho ... the Mac's serial number. I'll try to adjust the wording on the next commit to compensate (e.g., 'select the machine either via its plain name in the menu or its serial number in the menu').

**Comment by davidbritch** [comment]
I share your concerns. I noticed the weird path and changed back to the https://support.apple.com/guide/safari-developer/welcome/mac link. But then I didn't like the lack of content on that page and switched back to the weird path link.

I suspect it's better to go with the https://support.apple.com/guide/safari-developer/welcome/mac link, because the weird path segment makes me think it's produced by some kind of CMS and that the link might change the next time the page is updated.

**Review by davidbritch — APPROVED** [review]
LGTM!

**Comment by guardrex** [comment]
I totally agree, so I'll leave the link as-is with the updated link text.

Very well ... I'll shoot this to `main` and take it live right now. Thanks for your help with this.

**Pull request merged**